dts/bindings/spi/spi-controller.yaml

```diff
@@ -23,3 +24,33 @@ properties:
     cs-gpios:
       type: phandle-array
       required: false
+      description: |
+        An array of chip select GPIOs to use. Each element
+        in the array specifies a GPIO. The index in the array
+        corresponds to the child node that the CS gpio controls.
+
+        Example:
+
+          spi@... {
+                  cs-gpios = <&gpio0 pin0 flags0>,
```

Comment:
If you make this actual values rather than symbols, e.g. `<&gpio0 23 GPIO_ACTIVE_LOW>` that may be more helpful.  Otherwise or in addition I think it's worth a comment saying that the flags value should almost certainly be `GPIO_ACTIVE_LOW` unless intervening hardware inverts the signal from what the peripheral expects.

Personal opinion, no change required.